site/docs/concat.md

```diff
@@ -5,5 +5,44 @@ title: Concatenating views
 permalink: /docs/concat.html
 ---
 
-TODO
+To place views side-by-side, Vega-Lite's [view compositon algebra](composition.html) provides operators for horizontal (`hconcat`) and vertical (`vconcat`) concatenation.
 
+If you concatenate similar views where the only difference is the field that is used in an encoding, use the [`repeat` operator](repeat.html).
+
+## Horizontal Concatenation
+
+Put multiple views into a row by putting the specs for each view into an array and assign it to the `hconcat` property.
+
+{: .suppress-error}
+```json
+{
+  "hconcat": [
+    ...  // Specifications
```

Comment:
We can concatenate arbitrary views. However, we can only layer single or layered views. I revised the docs concerning this point/